deltachat-ffi/deltachat.h

```diff
@@ -2854,12 +2854,14 @@ int dc_receive_backup (dc_context_t* context, const char* qr);
  * @param dir The directory to create the context-databases in.
  *     If the directory does not exist,
  *     dc_accounts_new() will try to create it.
+ * @param writable Whether the returned account manager is writable, e.g. with
+ *     dc_accounts_add_account().
```

Comment:
i would be more precise here, eg.

```suggestion
 *     dc_accounts_add_account() or dc_accounts_remove_account().
 *     Only once process can hold a reference to a writable account manager.
 *     The accounts themselves as returned eg. by dc_accounts_get_account()
 *     are not affected by this flag and are always writable.
```

what about dc_select_account()? would this require writable accounts? not sure if that is used by the share extension currently.

Reply:
Yes, dc_select_account() needs to write the accounts config, will add to documentation